ctrlf.el

```diff
@@ -129,6 +136,51 @@ I have literally no idea why this is needed.")
 Assume that S2 has the same properties throughout."
   (apply #'propertize s1 (text-properties-at 0 s2)))
 
+(defun ctrlf--regexp-translate (str)
+  "Translate the user inputted regular expression STR.
+A single space character is translated into \".*\", while N spaces (N >= 2) are
+translated in to N-1 spaces. Space or spaces escaped with \"\\\" will not be
+translated."
+  ;; Replace escaped N spaces with N+1 spaces without backslash. So we can use
+  ;; a unified processing later. We only deal with cases where there are an odd
+  ;; number of backslashed before the spaces, so escaped backslashes are not
+  ;; affected.
+  (let* ((str (replace-regexp-in-string
+               "\\([^\\]\\|^\\)\\(\\\\\\\\\\)*\\\\\\( +\\)" "\\1\\2\\3 " str))
```

Comment:
This regexp has gotten to be pretty icky. What would you think of not trying to do anything magical, and instead offer exactly one way of inserting a literal space: type two spaces instead of one. That is very easy to understand and is unlikely to surprise anyone who uses a fuzzy search mechanism.

Reply:
Done.